certora/specs/Health.spec

```diff
@@ -75,6 +78,44 @@ filtered {
     assert !priceChanged => stillHealthy;
 }
 
+// The general case of liquidate for the stayHealthy rule
+//   - the user checked is not the borrower,
```

Comment:
Let's explain why it's sound (or not assume this, and make a branch)

Reply:
Actually this is verified to be true, with the `assert user != borrower` below. I changed the documentation